integrations/Axeptio/browser.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+import { NAME } from "./constants";
+import logger from "../../utils/logUtil";
+import { LOAD_ORIGIN } from "../ScriptLoader";
+import makeACall from "./utils";
+
+class Axeptio {
+  constructor(config) {
+    this.clientId = config.clientId;
+    this.name = NAME;
+  }
+
+  loadScript() {
+    window.axeptioSettings = {
+      clientId: this.clientId,
+    };
+    (function (d, s) {
+      var t = d.getElementsByTagName(s)[0],
+        e = d.createElement(s);
+      e.async = true;
+      e.src = "//static.axept.io/sdk.js";
+      e.setAttribute("data-loader", LOAD_ORIGIN),
+        t.parentNode.insertBefore(e, t);
+    })(document, "script");
+  }
+
+  init() {
+    logger.debug("===In init Axeptio===");
+    this.loadScript();
+  }
+
+  isLoaded() {
+    logger.debug("===In isLoaded Axeptio===");
+    return !!window.__axeptioSDK && typeof window.__axeptioSDK === "object";
+  }
+
+  isReady() {
+    logger.debug("===In isReady Axeptio===");
+    this.recordAxeptioEvents();
```

Comment:
We should provide an option in the dashboard to check if the customer wants to use the callback functionality.

Reply:
Yeah, we can do that. But, as the callback is the only supported feature here. Will it make sense?

Reply:
Yes. Someone may only want to load the axeptio SDK.

Reply:
Updated